sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClientDiagnostics.cs

```diff
@@ -32,28 +33,29 @@ public TextAnalyticsClientDiagnostics(ClientOptions options) : base(options)
             ResponseHeaders responseHeaders,
             ref IDictionary<string, string>? additionalInfo)
         {
-            if (!string.IsNullOrEmpty(content))
```

Comment:
curious about why was this causing a problem? 
can you create an issue to undo this?

Reply:
Fixed in PR https://github.com/Azure/azure-sdk-for-net/pull/28449